### PR TITLE
Added backend service faasd

### PR DESCRIPTION
**Pull request by ericm**
Added Dockerfile and compose entry for the `faasd` backend service.

### CODE REVIEW

**Review by Strum355 — COMMENTED** [review]
Does the Dockerfile work without the faasd repo cloned?

**Comment by ericm** [comment]
The Dockerfile clones the repo :)

File: backing-services/faasd/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Sourced from https://github.com/openfaas/faasd/pull/43/
+FROM golang:1.13-buster AS faasd
+ARG FAASD_VERSION=0.8.1
+
+RUN mkdir -p /go/src/github.com/openfaas/faasd
+RUN git clone -b ${FAASD_VERSION} --single-branch --depth 1 https://github.com/openfaas/faasd.git  /go/src/github.com/openfaas/faasd
```
Comment (ericm) [inline backing-services/faasd/Dockerfile:6]: Clones repo here

**Comment by Strum355** [comment]
Gonna leave this PR until its confirmed that we're going to be going this direction

**Pull request closed without merging**